README.md

```diff
@@ -267,6 +267,8 @@ While `nix-installer` tries to provide a comprehensive and unquirky experience,
 
 ### Using MacOS remote SSH builders, Nix binaries are not on `$PATH`
 
+*For Nix installations before Determinate Nix Installer version 0.14.1.*
```

Comment:
This part can be removed, we do not need to document old niche problems in the READMEs of new ones. I suspect we could look into cutting a new release quite soon, even.

Reply:
We might should keep a list of problems and quirks of old releases. Like the below note about nix-darwin should be solved now, now?

Reply:
The old releases have their own READMEs with quirk lists! :)

In the `nix-darwin` case, users may still try to uninstall Nix on their own before trying to use this tool, and we might get it wrong. We should be able to remove it if we implement a correct fix though.

Reply:
Cool!